### PR TITLE
Fixes #32455 - fix untranslated strings

### PR DESCRIPTION
**Pull request by Ron-Lavi**
sharing the findings of `eslint-plugin-translation`,
@jeremylenz could you take a look? thanks :)

### CODE REVIEW

File: webpack/scenes/ContentViews/Details/Repositories/LastSync.js
Comment (jeremylenz) [inline webpack/scenes/ContentViews/Details/Repositories/LastSync.js:10]: This will end up as a CSS style, so it shouldn't be translated

File: webpack/scenes/ContentViews/Copy/CopyContentViewModal.js
Comment (jeremylenz) [inline webpack/scenes/ContentViews/Copy/CopyContentViewModal.js:14]: `dangerouslySetInnerHTML` is bad, no?  Too bad we don't have any other way to translate text with formatting. :thinking:

File: webpack/scenes/ContentViews/Details/ComponentContentViews/ComponentVersion.js
Comment (jeremylenz) [inline webpack/scenes/ContentViews/Details/ComponentContentViews/ComponentVersion.js:23]: This will end up as a string like "Version 5.0", so "Version" alone might be hard to translate.  Not sure what to do about it though; should probably mark it for translation anyway.

File: webpack/scenes/Subscriptions/SubscriptionsPage.js
Comment (jeremylenz) [inline webpack/scenes/Subscriptions/SubscriptionsPage.js:277]: This is another partial that will be difficult to translate without context.

**Review by jeremylenz — CHANGES_REQUESTED** [review]
Thanks @laviro!

Should we also disable the eslint rule for the two unmodified files that github is complaining about?

File: webpack/scenes/ContentViews/Copy/CopyContentViewModal.js
Comment (laviro) [reply webpack/scenes/ContentViews/Copy/CopyContentViewModal.js:14]: yeah, we should investigate more about it,
though for now, I think it's the only solution...

File: webpack/scenes/ContentViews/Details/ComponentContentViews/ComponentVersion.js
Comment (laviro) [reply webpack/scenes/ContentViews/Details/ComponentContentViews/ComponentVersion.js]: using `sprintf`, although for `Version X` I don't know how many option there are.

File: webpack/scenes/Subscriptions/SubscriptionsPage.js
Comment (laviro) [reply webpack/scenes/Subscriptions/SubscriptionsPage.js]: Using dangerous sprintf here :D

File: webpack/scenes/ContentViews/Details/Repositories/LastSync.js
Comment (laviro) [reply webpack/scenes/ContentViews/Details/Repositories/LastSync.js]: in french it could have a nice sound: the new css `noir` :D

File: webpack/scenes/Subscriptions/SubscriptionsPage.js
Comment (jeremylenz) [reply webpack/scenes/Subscriptions/SubscriptionsPage.js]: What does dangerous sprintf mean?

Comment (laviro) [reply webpack/scenes/Subscriptions/SubscriptionsPage.js]: just kidding, I used the `sprintf` method from core and injected it with `dangerouslySetInnerHTML` to keep the html tags

File: webpack/scenes/ContentViews/Details/Histories/ContentViewHistories.js
Comment (laviro) [reply webpack/scenes/ContentViews/Details/Histories/ContentViewHistories.js:62]: well the eslint rule, for now, catches JSX strings and variables declaration
next thing would be to catch also strings like this one, but maybe first we should change it into the syntax of `return 'string'` without the braces 👼🏼

File: webpack/scenes/ContentViews/Details/ComponentContentViews/ComponentVersion.js
Comment (jeremylenz) [reply webpack/scenes/ContentViews/Details/ComponentContentViews/ComponentVersion.js]: It's the number of content view versions the user creates; I'd say it's usually between 1 and 20

**Review by jeremylenz — APPROVED** [review]
@laviro Needs a rebase but otherwise LGTM :+1:

**Comment by jeremylenz** [comment]
@laviro what's the status here?

**Comment by laviro** [comment]
Hey @jeremylenz, thanks for pinging me,
Looks like I will need to re-write the eslint package structure to modern structure because it does issues in other plugins, but we can use its suggestions and keep them in this PR

**Comment by jeremylenz** [comment]
@laviro is this PR still valid?

**Comment by Ron-Lavi** [comment]
updated after a long time :)
Ready for another round @jeremylenz

**Review by jeremylenz — COMMENTED** [review]
Thanks for updating @Ron-Lavi!

@amirfefer has alerted us to a way to deal with formatting in translations that I think is better than `dangerouslySetInnerHTML`, could you take a look?  https://github.com/theforeman/foreman/pull/8837/files

**Comment by Ron-Lavi** [comment]
@jeremylenz ready for another round

**Review by jeremylenz — APPROVED** [review]
Thanks @Ron-Lavi 

everything works as expected and looks good!

I would suggest changing the name of the PR before merging, since we didn't actually add the eslint plugin.

**Comment by Ron-Lavi** [comment]
done @jeremylenz

**Pull request merged**
